docs/UserGuide.md

```diff
@@ -74,6 +83,65 @@ Noted. I've removed this task:
 	Now you have 0 tasks in the list.
 ```
 
+###Listing tasks: `list`
+
+####Listing all tasks
```

Comment:
btw, I think you need a gap between the # and the words, i.e. `### Listing tasks: `list`` instead of `###Listing tasks: `list``, in order to make that a header.